package.json

```diff
@@ -67,13 +67,18 @@
     }
   },
   "dependencies": {
-    "bootstrap": "~3.3.6",
+    "bluebird": "^3.5.1",
+    "bootstrap": "~4.1.1",
     "dateformat": "^3.0.3",
     "electron-compile": "^6.4.2",
+    "electron-default-menu": "^1.0.1",
     "electron-devtools-installer": "^2.1.0",
+    "electron-fetch": "^1.2.0",
```

Comment:
:art: we just use `window.fetch` in GitHub Desktop rather than pulling in an extra dependency - I'm not sure of what else this offers, especially if this is in the renderer process 🤷‍♂️

Reply:
I'm using electron-fetch to get the access token in the main process in the OAuth flow. Not sure if I can use some default package but this made it pretty easy so I just went with it! Here's the code snippet https://github.com/education/classroom-desktop/blob/b5b3c0345fc992e5174f286f0217b766d413f85b/app/userAuthentication.js#L24-L31